www/app/lib/Zipfile.scala

```diff
@@ -0,0 +1,32 @@
+package lib
+
+import java.io.FileOutputStream
+import java.util.zip.{ZipEntry, ZipOutputStream}
+import com.bryzek.apidoc.generator.v0.models.File
+
+object Zipfile {
+
+  private[this] val UTF8 = "UTF-8"
+
+  def create(dirName: String, files: Seq[File]): java.io.File = {
+    val path = java.io.File.createTempFile(dirName, ".zip")
+    create(path, files, prefix = dirName + "/")
+    path
+  }
+
+  def create(
+    zip: java.io.File,
+    files: Seq[File],
+    prefix: String = ""
+  ) {
+    val zipOutputStream = new ZipOutputStream(new FileOutputStream(zip))
+    files.foreach { f =>
+      zipOutputStream.putNextEntry(new ZipEntry(s"$prefix${f.name}"))
```

Comment:
Actually dir is optional so it'll have to be:

    val path = prefix + f.dir.fold("")(_ + "/")
    new ZipEntry(s"$path${f.name}")